clio-dataaccess-model/src/main/scala/org/broadinstitute/clio/server/dataaccess/elasticsearch/DocumentGvcf.scala

```diff
@@ -17,7 +17,8 @@ case class DocumentGvcf(upsertId: UpsertId,
                         gvcfPath: Option[String] = None,
                         gvcfSize: Option[Long] = None,
                         gvcfIndexPath: Option[String] = None,
-                        gvcfMetricsPath: Option[String] = None,
+                        gvcfSummaryMetricsPath: Option[String] = None,
```

Comment:
I wonder if we should call this `variantCallingSummaryMetricsPath` instead, to match the extension.

Reply:
And same for the detail metrics.

Reply:
Unless there's any other gvcf metrics besides variant calling metrics, I think its ok. Probably should get @ktibbett thoughts on it.

Reply:
If we have to add another metric later we will have to reindex anyway so we could change the name then.. but it might be easier to just change it to be more specific now.